DigraphyApi/Dto/OrderDto.cs

```diff
@@ -0,0 +1,14 @@
+namespace DigraphyApi.Dto;
+
+public class OrderDto : CreateOrderDto
+{
+    public int Id { get; set; }
+}
+
+public class CreateOrderDto
+{
+    public string Name { get; set; }
+    public bool IsVerified { get; set; }
+    public DateTime CreatedAtUtc { get; set; }
+    public DateTime EditedAtUtc { get; set; }
```

Comment:
These should be part of the `OrderDto` rather than the `CreateOrderDto`. This is because we should set them on the server side after mapping it to a `Order`. We really don't want someone else being able to control these values.